api/graphql/schema.graphql

```diff
@@ -869,6 +873,7 @@ type CommunityQuerySet {
 type Skill {
   id: ID
   name: String
+  type: Int
```

Comment:
I would keep the `type` out as it's not being used. If a differentiation is needed in type then I'd carry through treating this as a separate type and make a `SkillToLearn` and `SkillsToLearnQuerySet`.